### PR TITLE
chore(deps, react-native): update to 0.72.14

### PR DESCRIPTION
**Pull request by Calinteodor**

### CODE REVIEW

File: ios/app/PrivacyInfo.xcprivacy [hunk 1]
```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
```
Comment (saghul) [inline ios/app/PrivacyInfo.xcprivacy:1]: We need to put this into the SDK instead.

File: ios/sdk/sdk.xcodeproj/project.pbxproj [hunk 2]
```diff
@@ -779,11 +779,7 @@
 				ONLY_ACTIVE_ARCH = YES;
 				OTHER_CFLAGS = "$(inherited)";
 				OTHER_CPLUSPLUSFLAGS = "$(inherited)";
-				OTHER_LDFLAGS = (
-					"$(inherited)",
-					"-Wl",
-					"-ld_classic",
-				);
+				OTHER_LDFLAGS = "$(inherited)";
```
Comment (saghul) [inline ios/sdk/sdk.xcodeproj/project.pbxproj:782]: What Xcode version and CocoaPods version are you using?

File: package.json [hunk 3]
```diff
@@ -36,7 +36,6 @@
     "@react-native-community/netinfo": "11.1.0",
     "@react-native-community/slider": "4.4.3",
     "@react-native-google-signin/google-signin": "10.1.0",
-    "@react-native/metro-config": "0.72.9",
```
Comment (saghul) [inline package.json:39]: This is the other PR right?

File: package.json [hunk 4]
```diff
@@ -82,7 +81,7 @@
     "react-focus-on": "3.8.1",
     "react-i18next": "10.11.4",
     "react-linkify": "1.0.0-alpha",
-    "react-native": "0.72.9",
+    "react-native": "0.72.14",
```
Comment (saghul) [inline package.json:84]: Nice didn't know they had made so many releases!

File: package.json (hunk 3 above)
Comment (Calinteodor) [reply package.json:39]: Yes. Will rebase.

File: ios/sdk/sdk.xcodeproj/project.pbxproj (hunk 2 above)
Comment (Calinteodor) [reply ios/sdk/sdk.xcodeproj/project.pbxproj:782]: 15.3 and 1.15.2

File: ios/app/PrivacyInfo.xcprivacy (hunk 1 above)
Comment (Calinteodor) [reply ios/app/PrivacyInfo.xcprivacy]: Moved.

File: ios/sdk/sdk.xcodeproj/project.pbxproj [hunk 5]
```diff
@@ -145,6 +145,7 @@
 		4ED4FFF12721B9B90074E620 /* JitsiAudioSession.h */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.h; path = JitsiAudioSession.h; sourceTree = "<group>"; };
 		4ED4FFF22721B9B90074E620 /* JitsiAudioSession.m */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.objc; path = JitsiAudioSession.m; sourceTree = "<group>"; };
 		4ED4FFF52721BAE10074E620 /* JitsiAudioSession+Private.h */ = {isa = PBXFileReference; lastKnownFileType = sourcecode.c.h; path = "JitsiAudioSession+Private.h"; sourceTree = "<group>"; };
+		6132EF172BDFF13200BBE14D /* PrivacyInfo.xcprivacy */ = {isa = PBXFileReference; lastKnownFileType = text.xml; name = PrivacyInfo.xcprivacy; path = ../PrivacyInfo.xcprivacy; sourceTree = "<group>"; };
```
Comment (Calinteodor) [inline ios/sdk/sdk.xcodeproj/project.pbxproj:148]: @saghul I think changes here will take care of both sdk and sdk lite, at least in the Podfile they both point to the same file.

**Review by saghul — DISMISSED** [review]
🚀

**Comment by saghul** [comment]
The CI is unhappy... looks related to the Xcode 15.3 changes I think.

**Review by saghul — APPROVED** [review]

**Comment by Calinteodor** [comment]
> The CI is unhappy... looks related to the Xcode 15.3 changes I think.

CI checks succeeded.

**Pull request merged**